rialto_airflow/funders/239_US_Federal_Funders_for_filter_with_ror.csv

```diff
@@ -0,0 +1,240 @@
+ID,Name,ror
```

Comment:
how about lowercasing all the headers?

Reply:
I guess now that we're not using the original dataset exactly as it was, this would make sense (and be more predictable). Will do.